api/lib/infrastructure/repositories/organization-invitation-repository.js

```diff
@@ -1,58 +1,39 @@
-import { BookshelfOrganizationInvitation } from '../orm-models/OrganizationInvitation.js';
-import * as bookshelfToDomainConverter from '../utils/bookshelf-to-domain-converter.js';
 import { NotFoundError } from '../../domain/errors.js';
 import { OrganizationInvitation } from '../../domain/models/OrganizationInvitation.js';
 import { knex } from '../../../db/knex-database-connection.js';
 import _ from 'lodash';
 
-function _toDomain(bookshelfInvitation) {
-  if (bookshelfInvitation) {
-    return bookshelfToDomainConverter.buildDomainObject(BookshelfOrganizationInvitation, bookshelfInvitation);
-  }
-  return null;
-}
-
-function _checkNotFoundError(error, id) {
-  if (error instanceof BookshelfOrganizationInvitation.NotFoundError) {
-    throw new NotFoundError(`Not found organization-invitation for ID ${id}`);
-  }
-  throw error;
-}
-
-function _checkNotFoundErrorWithCode({ error, id, code }) {
-  if (error instanceof BookshelfOrganizationInvitation.NotFoundError) {
-    throw new NotFoundError(`Not found organization-invitation for ID ${id} and code ${code}`);
-  }
-  throw error;
-}
-
-const create = function ({ organizationId, email, code, role }) {
+const create = async function ({ organizationId, email, code, role }) {
   const status = OrganizationInvitation.StatusType.PENDING;
-  return new BookshelfOrganizationInvitation({ organizationId, email, status, code, role }).save().then(_toDomain);
+  const [organizationInvitationCreated] = await knex('organization-invitations')
+    .insert({ organizationId, email, status, code, role })
+    .returning('*');
+  if (!organizationInvitationCreated) return null;
```

Comment:
En y réfléchissant, il me semble qu'on devrait ici tout simplement ne pas tester `organizationInvitationCreated`. En effet cette création d'enregistrement n'a pas de raison de produire un _Falsy_. Et s'il y a une erreur, ce serait une erreur globale, comme par exemple _« PostgreSQL ne peut plus écrire sur le système de fichiers car il n'y a plus de place »,_ qui serait alors récupérée par un `catch` plus haut dans la pile d'exécution et transformée plus loin comme il se doit en `500`.

```suggestion
  const [organizationInvitationCreated] = await knex('organization-invitations')
    .insert({ organizationId, email, status, code, role })
    .returning('*');
```